examples/winrm/microsoft_exchange.json

```diff
@@ -0,0 +1,35 @@
+{
+    "logo": "",
+    "version": "1.0.0",
+    "name": "Microsoft Exchange Server (on-prem)",
+    "description": "This script is used to monitor an on-prem Microsoft Exchange Server",
+    "category": "Software Integration",
+    "requirements": {
+        "credentials": true,
+        "sandbox_version": "1.12",
+        "others": [ 
+            "Powershell Version 5.1.19041.2364",
+            "WinRM Enabled"
+        ]
+    },
+    "tags": [
+        "windows",
+        "instance-name",
+        "exchange",
+        "email-server"
+    ],
+    "sample_period": 300,
+    "has_actions": false,
+    "expected_variables": {
+        "table": "[152]"
```

Comment:
This likely needs to be 
[48,999]

16 (the counters) * 3 (the table columns) 
Multiplied by all instances